packages/std/src/imports.rs

```diff
@@ -178,6 +182,27 @@ impl Api for ExternalApi {
         Ok(address.into())
     }
 
+    fn secp256k1_verify(
+        &self,
+        message_hash: &[u8],
+        signature: &[u8],
+        public_key: &[u8],
+    ) -> StdResult<bool> {
+        let hash_send = build_region(message_hash);
+        let hash_send_ptr = &*hash_send as *const Region as u32;
+        let sig_send = build_region(signature);
+        let sig_send_ptr = &*sig_send as *const Region as u32;
+        let pubkey_send = build_region(public_key);
+        let pubkey_send_ptr = &*pubkey_send as *const Region as u32;
+
+        let result = unsafe { secp256k1_verify(hash_send_ptr, sig_send_ptr, pubkey_send_ptr) };
+        match result {
+            1 => Ok(true),
+            0 => Ok(false),
+            x => panic!(format!("unexpected secp256k1_verify return value: {}", x)),
```

Comment:
```suggestion
            _ => unreachable!(),
```

We cat't extract the error messages of panics anyways. No need to generate Wasm code for this.